Library/Formula/postgresql.rb

```diff
@@ -77,8 +82,16 @@ def install
   end
 
   def post_install
-    unless File.exist? "#{var}/postgres"
-      system "#{bin}/initdb", "#{var}/postgres"
+    system "pkill postgres || true"
```

Comment:
I'll remove it here nonetheless. Appreciate it's not something we particularly want to catch on. ` Fish `, I *believe*, was done on security grounds on the last release, but we should be able to kill it on the next release, hopefully.